spec/inputs/jdbc_spec.rb

```diff
@@ -506,4 +506,39 @@
     end
   end
 
+  describe "config option force_lowercase_keys behaviour" do
+    let(:settings) { { "statement" => "SELECT * FROM ttt" } }
+    let(:events) { [] }
+
+    before do
+      db.create_table(:ttt) do
+        Integer(:num)
+        String(:somestring)
+      end
+      db[:ttt].insert(:num => 42, :somestring => "This is a string")
+      plugin.register
+    end
+
+    after do
+      plugin.stop
+      db.drop_table(:ttt)
+    end
+
+    context "when force_lowercase_keys is off (default)" do
```

Comment:
line 527: `    context "when force_lowercase_keys is off (default)" do`

line 535: `    context "when force_lowercase_keys is off" do`